ts/packages/ts/src/types/types.test.ts

```diff
@@ -17,6 +17,22 @@ describe(`types`, () => {
     });
   });
 
+  describe(`Cfgu`, () => {
+    const errored = { type: 'Test' };
+    it(`throw from Convert.toCfgu`, async () => {
+      const res = () => {
+        Convert.toCfgu(JSON.stringify(errored));
+      };
+      expect(res).toThrow();
+    });
+    it(`throw from Convert.cfguToJson`, async () => {
+      const res = () => {
+        Convert.cfguToJson(errored as any);
+      };
+      expect(res).toThrow();
+    });
+  });
```

Comment:
I must admit personally I don't _really_ see the value in testing autogenerated code directly like this - essentially this is duplicating tests that are (or should be) part of the `quicktype` package. I think it's better to assume these work as follows and test things that use this autogenerated code instead. 
Sort of follows the "test public code interfaces, not private ones"

That said, there's also no harm in having them here, so it's more a matter of taste

Reply:
I appreciate your perspective on testing autogenerated code. While I see the merit in focusing on public code interfaces, I opted to include basic error tests for the methods we actively use as a safeguard against potential issues with quicktype, such as unexpected bugs or breaking changes. Even though I'm keeping them for now, I fully agree that our project's test coverage can be improved and it's high time to develop more substantial tests.